# Request
bool isUpdate
---
# Response
uint8 length
value_desc[] values
